resources/views/community/components/developer/sets-page.blade.php

```diff
@@ -0,0 +1,153 @@
+@props([
+    'user' => null,
+    'consoles' => [],
+    'games' => [],
+    'sortOrder' => 'title',
+    'filterOptions' => [],
+    'userProgress' => null,
+])
+
+<x-app-layout
+    pageTitle="{{ $user->User }} - Developed Sets"
+    pageDescription="Games developed by {{ $user->User }}"
+>
+    <x-user.breadcrumbs :targetUsername="$user->User" currentPage="Developed Sets" />
+
+    <div class="mt-3 -mb-3 w-full flex gap-x-3">
+        {!! userAvatar($user->User, label: false, iconSize: 48, iconClass: 'rounded-sm') !!}
+        <h1 class="mt-[10px] w-full">{{ $user->User }}'s Developed Sets</h1>
+    </div>
+
+    <x-developer.sets-meta-panel
+        :selectedSortOrder="$sortOrder"
+        :filterOptions="$filterOptions"
+    />
+
+    @if (count($consoles) < 1)
+        <p>No developed games.</p>
+    @else
+        @foreach ($consoles as $console)
+            @if ($filterOptions['console'])
+                <h2 class="flex gap-x-2 items-center text-h3">
+                    <img src="{{ getSystemIconUrl($console->ID) }}" alt="Console icon" width="24" height="24">
+                    <span>{{ $console->Name }}</span>
+                </h2>
+            @endif
+
+            <div><table class='table-highlight mb-4'><tbody>
+
+            <tr>
+                <th style='width:34%'>Title</th>
+                <th style='width:12%' class='text-right'>Achievements</th>
+                <th style='width:10%' class='text-right'>Points</th>
+                <th style='width:10%' class='text-right'>RetroRatio</th>
+                <th style='width:10%' class='text-right'>Leaderboards</th>
+                <th style='width:8%' class='text-right'>Players</th>
+                <th style='width:8%' class='text-right'>Tickets</th>
+                @if ($userProgress !== null)
+                    <th style='width:8%' class='text-right'>Progress</th>
```

Comment:
It took me a second to understand whose or what progress this is (mine?, the developer I'm looking at?, the set's playerbase?, etc).

This column is fun and I like it, but because it took me a second to fully understand I'm afraid it leans towards being ambiguous. I know there is an aversion to putting labels like "Your" or "Mine" on things, but it does feel appropriate here to reduce that ambiguity.

Reply:
It's the logged in user's progress, and is meant to be propagated to pretty much every game list in the site as a replacement for the color script. As we haven't standardized how we want that to be displayed, I'm fine with removing it from this PR.

Reply:
I think it's neat and enhances the page. This minor ambiguity thing is the only thing about it that may need a slight change if it stays (my initial assumption was the bar represented the average progress of all players on the set).

Reply:
Would adding a tooltip help?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/bf1dff6d-c123-428d-8e6d-546b7dcef3ea)

Reply:
Yep, I think so. Perhaps on the column title label as well?